src/profile_manager/views.py

```diff
@@ -43,9 +55,15 @@ def queryset_append(self, profiles_qs):
         return profiles_qs
 
     def get_queryset(self):
-        profiles_qs = Profile.objects.all_students()
+        profiles_qs = Profile.objects.all_students().filter(user__is_active=True)
```

Comment:
I think a better way to do this is create a new Profile manager method all_inactive().  Then you can test the manager method directly, and if we use it elsewhere it's already tested and consistent.  Prefer to minimize filtering outside of manager methods, so we can reuse common/obvious filters (DRY!).

Also, this should be ALL inactive users, not just inactive students.

Ditto for other filters below.

Reply:
added function, but is there really a need to test a single arg filter?
![image](https://user-images.githubusercontent.com/39788517/183833969-d775108a-8f11-4939-8f11-2891ea7fa4b4.png)
![image](https://user-images.githubusercontent.com/39788517/183834358-5b9a4048-caf7-4672-bc98-b4a38d4bc23c.png)